cli/cmd/snapshot_test.go

```diff
@@ -188,6 +188,8 @@ func ExampleServicedCLI_CmdSnapshotAdd_usage() {
 	//    --description, -d 	a description of the snapshot
 }
 
```

Comment:
go test doesn't like when something exits non-zero, so it does not appear to be possible to run these tests successfully. However, there are now smoke tests that cover this behavior.